tests/app/api/business/brief/test_edit_opportunity.py

```diff
@@ -457,3 +618,72 @@ def test_edit_opportunity_creates_audit_event(self, agency_service, briefs, user
         ).one_or_none()
 
         assert audit_event is not None
+
+    def test_edit_attachments_updates_attachments(self, briefs):
+        specialist_brief = brief_service.get(2)
+        attachments = ['1.pdf', '2.pdf']
+
+        brief = brief_edit_business.edit_attachments(specialist_brief, attachments)
+        assert brief.data['attachments'] == attachments
+
+    def test_edit_response_template_updates_response_template(self, briefs):
+        rfx_brief = brief_service.get(3)
+        response_template = ['new.pdf']
+
+        brief = brief_edit_business.edit_response_template(rfx_brief, response_template)
+        assert brief.data['responseTemplate'] == response_template
+
+    def test_edit_empty_response_template_does_not_update_response_template(self, briefs):
+        rfx_brief = brief_service.get(3)
+        origin_template_template = rfx_brief.data['responseTemplate']
```

Comment:
`origin_template` or `original_template`?  There's another reference to this in a test below.